tests/sqllogictest/build.rs

```diff
@@ -9,8 +9,8 @@ fn main() {
     println!("cargo:rerun-if-changed=../../tests/sql");
 
     const PATTERN: &str = "../../tests/sql/**/[!_]*.slt"; // ignore files start with '_'
-    const MEM_BLOCKLIST: &[&str] = &["statistics.slt"];
-    const DISK_BLOCKLIST: &[&str] = &[];
+    const MEM_BLOCKLIST: &[&str] = &["statistics.slt", "join.slt"];
```

Comment:
If a test file is ignored for both cases, consider renaming it to `_join.slt` instead of changing the blocklist. Therefore, developers will be sure that it is disabled.

Reply:
Get it!